src/main/java/game/GameRunner.java

```diff
@@ -1,16 +1,17 @@
 package game;
```

Comment:
I found it a bit hard to find the main method when I was looking at your code. Should the `GameRunner` class, which is the entry point of the whole program, be outside the packages? Is that something you do in Java or am I imagining this? @gemcfadyen (Gabi, I don't know, asking Georgina for her opinion here...)

Reply:
Usually I would have a top level package, like rockPaperScissors. 
Then under that I would have game/   move/  player/   ui/

eg: 
```

rockPaperScissors/game/
rockPaperScissors/player/
```

This means your 'main' runner can sit under 'rockPaperScissors'
```
rockPaperScissors/GameRunner.java